modules/sdk-coin-coreum/test/unit/coreum.ts

```diff
@@ -1,20 +1,24 @@
+import { CosmosTransaction, SendMessage } from '@bitgo/abstract-cosmos';
 import { BitGoAPI } from '@bitgo/sdk-api';
-import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
```

Comment:
@hitansh-bitgo  Are we not using it ?

Reply:
We're still using it, the linter just sorted the imports